### PR TITLE
DOCS-4447: Update SITE information with GROUPS

### PR DESCRIPTION
**Pull request by JesseSnnt**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: api/multisafepay-api/listtransactions.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ slug: listtransactions
 ## How to test
 
 1. Under **Authentication** > **Query**:
-   - Enter your **TEST** account API key to list all transactions for your **account**.
-   - Enter the relevant site [API key](/docs/sites#site-id-api-key-and-security-code) from your **TEST** account to list all transactions for your **website** or **terminal groups**.
+   - Enter your **TEST** account API key to list all transactions for your **account**. This API key will be provided by your MultiSafepay sales agent.
```
Comment (inesmayerMSP) [inline api/multisafepay-api/listtransactions.md:10]: check wording

File: content/getting-started/sites.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ To view the ID, API key, and security code for a website:
 
 1. Sign in to your <a href="https://merchant.multisafepay.com" target="_blank">live</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> or <a href="https://testmerchant.multisafepay.com" target="_blank">test</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i> MultiSafepay dashboard.
 2. Go to **Websites**, and then click on the relevant website.
-3. The website ID, API key, and security code appear in the top-right corner of your profile.
+3. You can find the website ID, API key, and security code in the top-right corner of the page.
```
Comment (inesmayerMSP) [inline content/getting-started/sites.md:136]: where is the group api key

File: content/payment-management/webhook.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ For websites:
 
 1. Sign in to your <a href="https://merchant.multisafepay.com" target="_blank">MultiSafepay dashboard</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>.
 2. Go to **Websites**, and then click the relevant website.
-3. On the **Website profile** page, under **Functionality**, in the **Webhook URL** field, set your webhook endpoint.
+3. On the **Website profile** page, under **Functionality**, in the **Webhook URL** field, set your webhook endpoint.
```
Comment (inesmayerMSP) [inline content/payment-management/webhook.md:58]: lots of instructions

> >

**Review by inesmayerMSP — CHANGES_REQUESTED** [review]

File: content/getting-started/sites.md (hunk 2 above)
Comment (JesseSnnt) [reply content/getting-started/sites.md:136]: this is in another commit, below the "To view the ID, API key, and security code for a website:" instructions.

**Pull request merged**